nautobot/extras/tests/test_models.py

```diff
@@ -1072,6 +1073,36 @@ def test_remote_url_hostname(self):
         self.repo.remote_url = "http://some-private-host/example.git"
         self.repo.validated_save()
 
+    def test_clone_and_discard_clone_functions(self):
+        """Confirm that the clone() and discard_clone() methods work as expected."""
+        self.repo.remote_url = "https://github.com/nautobot/nautobot.git"
+
+        with self.subTest("Clone a repository with no path argument provided"):
+            path = self.repo.clone()
+            # assert that the temporary directory was created in the expected location i.e. /tmp/
+            default_dir = "tmp"
+            self.assertIn(default_dir, path)
```

Comment:
Feels overly non-specific. We should assert that `path.startswith(tempfile.gettempdir()`.